torch/utils/cpp_extension.py

```diff
@@ -740,6 +763,21 @@ def _check_abi(self):
                    'Please set `DISTUTILS_USE_SDK=1` and try again.')
             raise UserWarning(msg)
 
+    def _check_cuda_version(self):
+        if CUDA_HOME:
+            nvcc = os.path.join(CUDA_HOME, 'bin', 'nvcc')
+            cuda_version_str = subprocess.check_output([nvcc, '--version']).strip().decode()
+            cuda_version = re.search(r'release (\d+[.]\d+)', cuda_version_str)
+            if cuda_version is not None:
+                cuda_str_version = cuda_version.group(1)
+                cuda_version = parse_version(cuda_str_version)
+                torch_cuda_version = parse_version(torch.version.cuda)
+                if cuda_version.major != torch_cuda_version.major:
```

Comment:
This is the first part of https://github.com/pytorch/pytorch/issues/55267#issuecomment-860859575. Shouldn't it be followed by
```python
elif cuda_version.minor != torch_cuda_version.minor:
    warnings.warn(....)
```
?